### PR TITLE
fix(build): move some helpers to makefile

### PR DESCRIPTION
**Pull request by fzipi**
- move additional helpers to makefile
- update README with the move

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,10 +94,26 @@ Copyright (c) 2012-2016 Nick Galbreath
 Licensed under the standard [BSD 3-Clause](http://opensource.org/licenses/BSD-3-Clause) open source
 license.  See [COPYING](/COPYING) for details.
 
+## BUILD TARGETS
+
+Some of the previous help runners have been merged into the Makefile. E.g.:
```
Comment (migueldemoura) [inline README.md:99]: Do we need to point this out?

File: README.md [hunk 2]
```diff
@@ -96,10 +94,26 @@ Copyright (c) 2012-2016 Nick Galbreath
...
+
+* run-clang-asan.sh -> `make clan-asan`
+* make-ci.sh -> `make ci`
```
Comment (migueldemoura) [inline README.md:102]: Do we still need these two scripts?

**Review by migueldemoura — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (fzipi) [reply README.md:99]: Well, I would start with this until we transition to completely removing them. I saw they were used here: https://github.com/libinjection/libinjection/pull/36, so maybe rethinking the documentation will be the best option in the end.

File: README.md (hunk 2 above)
Comment (fzipi) [reply README.md:102]: Probably not, but let's provide them until we overhaul docs.

**Pull request merged**